test/basic-querying.test.js

```diff
@@ -127,7 +127,23 @@ describe('basic-querying', function() {
 
   describe('find', function() {
 
-    before(seed);
+    before(function() {
+      seed(function setupDelayingLoadedHook() {
+        User.observe('loaded', function(ctx, next) {
+          var randomTimeoutTrigger = Math.floor(Math.random() * 100);
+
+          setTimeout(function() {
+            process.nextTick(next);
+          }, randomTimeoutTrigger);
+        });
+      });
+    });
+
+    after(function removeDelayingLoadHook() {
+      User.observe('loaded', function(ctx, next) {
```

Comment:
Edit: the observer API is included in our API docs, see http://apidocs.strongloop.com/loopback-datasource-juggler/#observermixin-removeobserver